### DISCUSSION TITLE
Argocd is keep resyncing the service.

### DISCUSSION THREAD

**Discussion by room303** [argoproj/argo-cd#13908] [Q&A]
Hey Guys,

I have installed my application in the argocd. One of the component is having KIND as job, it keeps re-syncing which is not required. When i configured self heal as false for this component it is running for one time and showing as out of sync. 

 when we have done the helm installation manually this service is getting deleted automatically. Do we have any method or configuration that this service has been synced and shows healthy in the argocd.? 

I will provide if any information is required. 

Thanks in advance.

**Comment by ashutosh16** [comment]
See if there is any annotations available - https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/
